environments/development/corp/fileloader/workflow.yml

```diff
@@ -0,0 +1,23 @@
+---
+dag:
+  repository: moj-analytical-services/ap-airflow-corp-fileloader
+  tag: 0.0.1
+
+maintainers:
+  - williamorrie
+
+notifications:
+  - dmet-corp-notifications
+
+athena: write
+
+iam:
+  s3_read_only:
+    - alpha-div-inc/sscl-sourced-gcdi/*
+  s3_write_only:
+    - mojap-processed/corporate/sop/dev-iceberg/*
+    - alpha-corp-data-staging/athena_queries/temp_data/athena_query_dump/*
+
+tags:
+  business_unit: DEDSAI
```

Comment:
`business_unit` must map to one of these https://technical-guidance.service.justice.gov.uk/documentation/standards/documenting-infrastructure-owners.html#mandatory, see https://github.com/ministryofjustice/analytical-platform-airflow/blob/main/scripts/workflow_schema_validation/schema.json#L191-L201